cmd/devstream/apply.go

```diff
@@ -22,7 +22,7 @@ func applyCMDFunc(cmd *cobra.Command, args []string) {
 
 	err := pluginengine.Apply(configFile)
 	if err != nil {
-		log.Printf("Apply error: %s.", err)
+		log.Printf("Apply error: %s. Maybe you forgot to run \"dtm init\" first?", err)
```

Comment:
What if `pluginmanager.CheckLocalPlugins(cfg)` return nil but some error occurred below in `Apply()`?

Reply:
Good point. This error message should be put in the `Apply()` func right after checking plugins. Will move it.